src/main/java/com/capitalone/dashboard/collector/PullRequestCollector.java

```diff
@@ -249,7 +262,7 @@ private void populateScmRevisionNumber(GitRequest pull, JSONObject jsonObject) {
         }
     }
 
-    private GitRequest getPullRequest(
+    private GitRequest getPullRequestServer(
```

Comment:
may need to change the name.
and have to investigate the response from bitbucket server